docs(ha): wording update when using redis (#2771)
runatlantis/atlantis: runatlantis.io/docs/faq.md

A: Atlantis server can easily be run under the supervision of a init system like `upstart` or `systemd` to make sure `atlantis server` is always running.

> Atlantis, by default, stores all locking and Terraform plans locally on disk under the `--data-dir` directory (defaults to `~/.atlantis`). If you are planning to run multiple Atlantis hosts (utilizing a shared redis backend), you will want to make sure that the `data-dir` is using a shared filesystem between hosts.

Reviewer: Better to drop pronouns

https://ohiostate.pressbooks.pub/feptechcomm/chapter/3-1-voice-tone/

>In scientific or technical writing, there is a common (but not universal) perception that personal pronouns can undermine the objectivity of the writer or distract from important information. This is one of the main reasons passive voice appears so frequently in scientific or technical writing—the focus is shifted away from the person doing the action.

```suggestion
Atlantis, by default, stores all locking and Terraform plans locally on disk under the `--data-dir` directory (defaults to `~/.atlantis`). If multiple Atlantis hosts are run by utilizing a shared redis backend, then it's important that the `data-dir` is using a shared filesystem between hosts.
```
